<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIMainWindow</class>
 <widget class="QMainWindow" name="UIMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageConcat</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>931</width>
      <height>511</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSplitter" name="top">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGraphicsView" name="imgDisplay"/>
     <widget class="QGroupBox" name="checkGroup">
      <property name="title">
       <string>Options</string>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="buttom">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="browseButton">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
